apps/web/src/components/healthcare/ClinicalWorkflowPatterns.tsx

```
@@ -0,0 +1,851 @@
+          {emergencyMode && (
+            <Badge variant="destructive" className="animate-pulse">
+              <Zap className="w-3 h-3 mr-1" aria-hidden="true" />
+              URGENTE
+            </Badge>
+          )}
+        </div>
+
+        {/* Appointment Type Filter */}
+        <div className="space-y-2">
+          <label className="text-sm font-medium text-[#112031]">Tipo de Consulta</label>
+          <div className="flex flex-wrap gap-2">
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)